widgets/executions/src/backend.ts

```diff
@@ -1,5 +1,44 @@
-// @ts-nocheck File not migrated fully to TS
-export default r => {
+import type { ElkNode } from 'elkjs';
+import type { BackendServiceRegistrator } from '../../../backend/handler/BackendHandler.types';
+import type { PaginatedResponse } from '../../../backend/types';
+
+declare module 'elkjs/lib/elk-api' {
+    interface ElkLabel {
+        displayTitle?: string[];
+        displayText?: string[];
+        state?: string;
+        retry?: number;
+        type?: string;
+        operation?: string;
+    }
+    // eslint-disable-next-line @typescript-eslint/no-shadow
+    interface ElkNode {
+        nodeInstanceId?: string;
+        operation?: string;
+        containingSubgraph?: string;
+    }
+}
+
+interface Operation {
+    id: string;
+    name?: string;
+    type?: string;
+    state?: string;
+    parameters: {
+        /* eslint-disable camelcase */
+        task_kwargs: {
+            cloudify_context?: { operation?: { name?: string } };
+            kwargs?: unknown;
+        };
+        containing_subgraph?: string;
+        current_retries: number;
+        retried_task?: unknown;
+        /* eslint-enable camelcase */
+    };
+    dependencies: unknown[];
+}
+
+export default (r: BackendServiceRegistrator) => {
     r.register('get_tasks_graph', 'GET', (req, res, next, helper) => {
```

Comment:
We could omit the usage of ominous letter in order of having meaningful variable names 💪 

```suggestion
export default (serviceRegistrator: BackendServiceRegistrator) => {
    serviceRegistrator.register('get_tasks_graph', 'GET', (req, res, next, helper) => {
```

I've double checked and it looks like that there are no more `r` occurences that would require adjustments from our side 🎉

Reply:
Done